app/config/local.php.backup

```diff
@@ -0,0 +1,786 @@
+<?php
```

Comment:
@mollux was this file merged by accident?

Reply:
indeed, fixed